pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/custom/GeoSpatialTest.java

```diff
@@ -234,6 +234,19 @@ public void testStPointLiteralFunction(boolean useMultiStageQueryEngine)
       byte[] actualValue = BytesUtils.toBytes(result);
       assertEquals(actualValue, expectedValue);
     }
+
+    for (int isGeography = 0; isGeography < 2; isGeography++) {
+      String query = String.format("Select ST_Point(116.407394,39.904202, %d) from %s", isGeography, getTableName());
```

Comment:
Not introduced in this PR, but we should probably put boolean is the third argument